source/common/version/BUILD

```diff
@@ -50,8 +50,8 @@ envoy_cc_library(
     name = "version_lib",
     srcs = ["version.cc"],
     copts = envoy_select_boringssl(
-        ["-DENVOY_SSL_VERSION=\\\"BoringSSL-FIPS\\\""],
-        ["-DENVOY_SSL_VERSION=\\\"BoringSSL\\\""],
+        ["-DENVOY_SSL_VERSION=\"BoringSSL-FIPS\""],
+        ["-DENVOY_SSL_VERSION=\"BoringSSL\""],
```

Comment:
this change may have broken CI, please take a look.

Reply:
Sorry my local tool chain occasionally spoil this. Fixing...